### DISCUSSION TITLE
Apex chart in community contributions

### DISCUSSION THREAD

**Discussion by t288matt** [BottlecapDave/HomeAssistant-OctopusEnergy#264] [Q&A] [answered]
Hi all, I can't get the indentation right on this apex chart from @plandregan in the community contribution section
https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/community.md

Has anyone got it working and can share please.

**Comment by BottlecapDave** [comment, marked as answer]
I copied the logic from https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/145, but never tried it myself. I'll ask in the ticket for clarification on formatting.

**Comment by BobC76** [comment]
Hi FWIW I only found this _after_ I'd worked out my own version having not found an answer elsewhere - see https://github.com/lozzd/octopus-energy-rates-card/issues/13#issuecomment-1638136012 for some alternative code
